src/canoc/canoc/transceiver.py

```diff
@@ -17,13 +17,16 @@ class Transceiver(Node):
 
     # These are the IDs to look for the relevant can buses
     RADAR_IDS_MATCH = [0x220, 0x221, 0x222, 0x223, 0x224, 0x225]
-    CONTROL_IDS_MATCH = [0x100, 0x101]
+    CONTROL_IDS_MATCH = [0x73, 0x83, 0x93]
```

Comment:
So the 0x100, 0x101 were just placeholders and the new values are the correct ones for the OSCC controller?

Reply:
yes